tests/methodologies/para/test_feature_extractor.py

```
@@ -579,26 +577,46 @@ def __init__(self, *, now: float) -> None:
                 self.st_mtime = now - (10 * 86400)
+                self.st_birthtime = now - (5 * 86400)  # macOS birth time
                 self.st_mode = 0o100644  # regular file — required by Path.is_dir()
 
         test_file = tmp_path / "test.txt"
         test_file.write_text("content")
         now = 2_000_000_000.0
 
-        # Mock os module in the feature_extractor namespace
-        monkeypatch.setattr(fe_module.os, "name", "nt")
         monkeypatch.setattr(fe_module.time, "time", lambda: now)
         monkeypatch.setattr(Path, "stat", lambda _self, *args, **kwargs: MockStat(now=now))
```

Ruff (0.15.9)

[warning] Unused lambda argument: `args`

(ARG005)